tests/.eslintrc

```diff
@@ -5,5 +5,9 @@
   },
```

Comment:
Huh, I just noticed `mocha: true` in `env`. You can remove that. That rule allowed us to reference `describe()` as a global but we never do that (we import it) so the rule is not necessary.